### PR TITLE
Added "emote" and moved the players into an embed yes

### PR DESCRIPTION
**Pull request by Apfel**

### CODE REVIEW

**Comment by Alej0hio** [comment]
please do not merge

**Comment by Apfel** [comment]
Gonna make this embedded i guess

File: src/main/java/tk/liblnd/bot/commands/discord/ListCmd.java [hunk 1]
```diff
@@ -39,7 +39,7 @@ public void execute(CommandEvent event)
 
         StringBuilder sb = new StringBuilder("Players online on LibertyLand:\n");
         for(ProxiedPlayer p : players)
-            sb.append("**").append(p.getName()).append("**, ");
+            sb.append(":white_medium_small_square: **").append(p.getName()).append("**, ");
```
Comment (Artuto) [inline src/main/java/tk/liblnd/bot/commands/discord/ListCmd.java]: This would add that emote at the start of every player name

Comment (Apfel) [reply src/main/java/tk/liblnd/bot/commands/discord/ListCmd.java]: that's intended

Comment (Artuto) [reply src/main/java/tk/liblnd/bot/commands/discord/ListCmd.java]: I dont really want it like that

Comment (Apfel) [reply src/main/java/tk/liblnd/bot/commands/discord/ListCmd.java]: Well i make this an embed anyways so gimme some time to make this better i guess

**Comment by Apfel** [comment]
i changed the "emote" heh

**Comment by Apfel** [comment]
Well this thing is finished i guess heh

**Comment by Apfel** [comment]
noice. Thank you!

**Pull request merged**